apps/voting/contracts/Voting.sol

```diff
@@ -58,14 +62,19 @@ contract Voting is IForwarder, AragonApp {
     mapping (uint256 => Vote) internal votes;
     uint256 public votesLength;
 
+    uint64 public overruleWindow;
```

Comment:
Shouldn't this be added to the `initialize` function?

Reply:
I did this way thinking of current deployed voting apps, assuming those are already initialized, it won't work, they should use the setter instead. But now that you mention it, it may be necessary for new instances of the voting app :+1:

Reply:
Wanted to bring this up again; was there a decision around adding it to the `initialize()` function?

I was thinking about if a default of 0 made sense, and my conclusion is that it does (so adding this to the `initialize()` function is somewhat optional and really depends on if we want to make using it in templates cheaper).